### PR TITLE
improved advanced docs 😎

### PR DESCRIPTION
**Pull request by balajmarius**
wait until animations end is way better than timeouts ⏰

### CODE REVIEW

**Comment by IonicaBizau** [comment]
I agree! Didn't know about that event! :+1:

File: README.md [hunk 1]
```diff
@@ -26,15 +26,18 @@ If you need more stuff (e.g. animations etc), you need to create custom handlers
...
-    setTimeout(function () {
-        self.box.classList.add("animated", "fadeOut");
-        setTimeout(function () {
-            self.box.classList.remove("animated", "fadeOut");
-            self.img.classList.remove("animated", "fadeOutDown");
-            self.box.style.display = "none";
-        }, 500);
-    }, 500);
+    // wait until animation end
+    self.img.addEventListener(animationEnd, function() {
```
Comment (IonicaBizau) [inline README.md]: However, we should `removeEventListener` after the function is executed. Otherwise, it's going to add new listeners every time.

**Comment by balajmarius** [comment]
updated 🖊

**Comment by balajmarius** [comment]
Done 😎.

**Pull request merged**